### ISSUE TITLE
Failed test 'Got a number'

### ISSUE THREAD

**Issue by eserte** [PopeFelix/Mojo-UserAgent-Mockable#27]
Sometimes the test suite fails on my smoker systems:
```
#   Failed test 'Got a number'
#   at t/record_playback_proxy.t line 44.
# Looks like you failed 1 test of 4.
t/record_playback_proxy.t ........... 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/4 subtests 
```
This seems to happen also on other systems: http://matrix.cpantesters.org/?dist=Mojo-UserAgent-Mockable%201.57

**Comment by kpeters-cbsi** [comment]
I think I need to get rid of all the tests that rely on external services and just spawn an HTTP server for testing or something. I suspect this problem is related to exceeding resource limits on random.org.

**Comment by PopeFelix** [comment]
Probably fixed in #33
